ci/jenkins/test-vmc.sh

```diff
@@ -179,6 +179,21 @@ function setup_cluster() {
         sed -i "s|${kv[0]}|${kv[1]}|g" ${GIT_CHECKOUT_DIR}/jenkins/out/cluster.yaml
     done < "$cluster_defaults"
 
+    # Count the number of clusters created with this network.
+    # If count equals to 6 (2 round of /test-all), switch another one.
```

Comment:
That works for me, but I'm curious as to why we do not use round-robin?

Reply:
Right. I used to think that since there are 2 networks for Antrea CI, there is no need for round-robin. But maybe let's get prepared more networks in the future. Updated.